spec.md

```diff
@@ -1080,6 +1081,135 @@ A dictionary value has these methods:
 * [`values`](#dict·values)
 
 
+### Sets
+
+A set is a mutable, iterable collection of unique values - the set's *elements*.
+The [type](#type) of a set is `"set"`.
+
+Sets provide constant-time operations to insert, remove, or check for the
+presence of a value. Sets are implemented using a hash table, and therefore,
+just like keys of a [dictionary](#dictionaries), elements of a set must be
+hashable. A value may be used as an element of a set if and only if it may be
+used as a key of a dictionary.
+
+Sets may be constructed using the [set()](#set) built-in function.
+
+The `in` and `not in` operations check whether a value is in a set:
+
+```python
+s = set(["a", "b", "c"])
+"a" in s  # True
+"z" in s  # False
+```
+
+A set is an iterable sequence. Its length can be retrieved using the
+[len()](#len) built-in function, and the order of iteration is the order in
+which elements were first added to the set:
+
+```python
+s = set(["z", "y", "z", "y"])
+len(s)       # prints 2
+s.add("x")
+len(s)       # prints 3
+for e in s:
+    print e  # prints "z", "y", "x"
+```
+
+A set used in Boolean context is true if and only if it contains one or more
+elements.
+
+```python
+s = set()
+"non-empty" if s else "empty"  # "empty"
+t = set(["x", "y"])
+"non-empty" if t else "empty"  # "non-empty"
+```
+
+Sets may be compared for equality using `==` and `!=`. A set `s` is equal to `t`
+if and only if `t` is a set containing elements equal to the elements of `s`,
+but possibly in a different iteration order. In particular, a set is *not* equal
+to the list of its elements. Sets are not ordered with respect to other sets,
+and an attempt to compare two sets using `<`, `<=`, `>`, `>=`, or to sort a
+sequence of sets, will fail.
+
+```python
+set() == set()              # True
+set() != []                 # True
+set([1, 2]) == set([2, 1])  # True
+set([1, 2]) != [1, 2]       # True
+```
+
+The `|` operation on two sets returns the union of the two sets: a set
+containing the elements found in either one or both of the original sets. The
+`|` operation has an augmented assignment version; `s |= t` adds to `s` all the
```

Comment:
The "augmented" business can be dealt with in a single sentence at the end; no need to state and restate the obvious.

Reply:
Good point!